CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## v0.26.2
+
+*November 15th, 2018*
+
+Special thanks to external contributors on this release: @hleb-albau, @zhuzeyu
```

Comment:
hmm.. i don't see anything from zhuzeyu below

Reply:
It's the one liner that replaced stateKey with key - I thought we decided some things aren't worth an entry but worth mentioning the contributor?